### PR TITLE
[1LP][RFR] Removed 99% of 5.4s and a few 5.5s

### PR DESCRIPTION
**Pull request by psav**
Removed 99% of 5.4s and a few 5.5s

### CODE REVIEW

File: cfme/cloud/provider/__init__.py [hunk 1]
```diff
@@ -48,7 +48,6 @@
         ('azure_tenant_id', Input("azure_tenant_id")),
         ('name_text', Input("name")),
         ('hostname_text', Input("hostname")),
-        ('ipaddress_text', Input("ipaddress"), {"removed_since": "5.4.0.0.15"}),
         ('region_select', {version.LOWEST: Select("select#provider_region"),
             "5.5": AngularSelect("provider_region")}),
```
Comment (mshriver) [inline cfme/cloud/provider/__init__.py]: Intentionally leaving this one?

File: cfme/web_ui/search.py [hunk 2]
```diff
@@ -16,10 +16,8 @@
         search_field=Input("search_text", "search[text]"),
 
         # The icon buttons for searching
-        search_icon={
-            "5.4":
-            "//div[@id='searchbox']//div[contains(@class, 'form-group')]"
-            "/*[self::a or (self::button and @type='submit')]"},
+        search_icon=("//div[@id='searchbox']//div[contains(@class, 'form-group')]"
```
Comment (mshriver) [inline cfme/web_ui/search.py]: **Optional**
I've got this one covered in #4180 with updated xpath if you want to leave it off here. No worries either way but you might have to merge my changes if that PR gets in first.

**Comment by izapolsk** [comment]
@psav, your changes look fine but some PRT tests have failed. Can you run thru them and make sure that failures aren't related to your changes ?

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request merged**